plugins/plugin-chart-echarts/src/MixedTimeseries/buildQuery.ts

```diff
@@ -61,45 +56,19 @@ export default function buildQuery(formData: QueryFormData) {
   };
 
   const queryContextA = buildQueryContext(formData1, baseQueryObject => {
-    const metricLabels = (baseQueryObject.metrics || []).map(getMetricLabel);
     const queryObjectA = {
       ...baseQueryObject,
       is_timeseries: true,
-      post_processing: [
-        {
-          operation: 'pivot',
-          options: {
-            index: ['__timestamp'],
-            columns: formData1.columns || [],
-            // Create 'dummy' sum aggregates to assign cell values in pivot table
-            aggregates: Object.fromEntries(
-              metricLabels.map(metric => [metric, { operator: 'sum' }]),
```

Comment:
`pivotOperator()`  uses `mean` rather than `sum`. Does these aggregates being "dummy" mean that it doesn't matter whether which operator we use?

Reply:
Yes, It does. The main reason for using `mean` is to avoid `drop NaN`. refer to: https://github.com/apache-superset/superset-ui/pull/1231